salt/utils/parsers.py

```diff
@@ -974,7 +974,7 @@ def _mixin_before_exit(self):
                 try:
                     os.unlink(self.config['pidfile'])
                 except OSError as err:
-                    self.info(
+                    logging.getLogger(__name__).info(
```

Comment:
Please don't. Typically we do:
```python
import logging
log = logging.getLogger(__name__)

...

log.info(....
```
I understand the entire `parsers.py` doesn't have logging yet except here, but this is subject to change.

Reply:
I can do that, but not in this PR. I've quickly looked through the code and there is already a  `log`, but I don't know yet where this is coming from and if this would clash. So let's just fix the bug here and do the refactoring in a second PR.

Reply:
After this PR is merged, I'll create a PR for [this](https://github.com/brejoc/salt/commit/4c51a4f08691324ea132cd3a5072fbfae3e02a76).